roles/openshift_health_checker/openshift_checks/package_update.py

```diff
@@ -9,6 +9,24 @@ class PackageUpdate(NotContainerizedMixin, OpenShiftCheck):
     name = "package_update"
     tags = ["preflight"]
 
+    supported = ["Red Hat Enterprise Linux"]
+
     def run(self, tmp, task_vars):
         args = {"packages": []}
-        return self.module_executor("check_yum_update", args, tmp, task_vars)
+
+        os = self.get_operating_system(task_vars)
+        if os and self.is_os_supported(os):
+            return self.module_executor("check_yum_update", args, tmp, task_vars)
+
+        return {"changed": False, "msg": "Skipping check on unsupported OS: {os}".format(os=os)}
+
+    def get_operating_system(self, task_vars):
+        info = self.module_executor("docker_info", {}, task_vars)
```

Comment:
```console
$ ansible -i hosts -m setup master1 | grep yum
        "ansible_pkg_mgr": "yum", 
```

All we need to do is implement `is_active` to look at `task_vars['ansible_pkg_mgr']`.